org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/format/experimental/XMLFormatterMaxLineWithTest.java

```diff
@@ -69,6 +69,129 @@ public void noSplit() throws BadLocationException {
 		assertFormat(content, expected, 20);
 	}
 
+	@Test
+	public void splitWithAttribute() throws BadLocationException {
+		SharedSettings settings = new SharedSettings();
+		settings.getFormattingSettings().setTabSize(4);

```

Comment:
My first impression when I see this code, is why this test doesn't configure maxLineWidth in the test although test tests this max line width. The answer is that it uses default value, but it is hard to understand that when you read the test. More if you change the default value for max line widg (ex : null), this test will not work again.

In otherwords, please define the max line width for each tests which tests max line width.

Reply:
The assertFormat method in this test takes in the maxLineWidth as an argument like this: 

`	private static void assertFormat(String unformatted, String expected, int maxLineWidth, TextEdit... expectedEdits)
`

but I can see how it is unclear, I can change it so that it is more explicitly defined if that makes more sense?

Reply:
Oh I see. I think to be more consistent with other test. You should update the max line with in shared settings like tab size